README.md

```diff
@@ -317,14 +318,18 @@ structs to the `Scan()` function:
 For two or three dimensional arrays, use `trino.NullSlice2Bool` and
 `trino.NullSlice3Bool` or equivalents for other data types.
 
+`VARBINARY` columns are returned as base64-encoded strings when used within
+`ROW`, `MAP`, or `ARRAY` values.
+
 To read `ROW` values, implement the `sql.Scanner` interface in a struct. Its
 `Scan()` function receives a `[]interface{}` slice, with values of the
 following types:
 * `bool`
 * `json.Number` for any numeric Trino types
 * `[]interface{}` for Trino arrays
 * `map[string]interface{}` for Trino maps
-* `string` for other Trino types, as character, date, time, or timestamp
+* `string` for other Trino types, as character, date, time, or timestamp.
+  * `VARBINARY` columns are returned as base64-encoded strings
```

Comment:
Is this supposed to be a one item sub-list? It looks weird, maybe add it as a note below the list.